jbpm-services/jbpm-executor/data/journal/server.lock

```diff
@@ -0,0 +1 @@
+P00�Ѻ�;��ڹL�:
```

Comment:
github support: adding a line comment is possible. 

However, the "pencil" icon in the upper right hand corner is showing a message that I'm not able to add a comment unless I'm signed in and have push access. 

My further apologies for the vagueness.. :/